frontend/packages/dev-console/src/components/helm/HelmReleaseRollbackPage.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { Formik } from 'formik';
+import { RouteComponentProps } from 'react-router';
+import { PageBody } from '@console/shared';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { PageHeading, history } from '@console/internal/components/utils';
+
+import { HelmRelease } from './helm-types';
+import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
+import HelmReleaseRollbackForm from './form/HelmReleaseRollbackForm';
+
+type HelmReleaseRollbackPageProps = RouteComponentProps<{
+  ns?: string;
+  releaseName?: string;
+}>;
+
+type HelmRollbackFormData = {
+  revision: number;
+};
+
+const HelmReleaseRollbackPage: React.FC<HelmReleaseRollbackPageProps> = ({ match }) => {
+  const { releaseName, ns: namespace } = match.params;
+  const [releaseHistory, setReleaseHistory] = React.useState<HelmRelease[]>(null);
+
+  React.useEffect(() => {
+    let ignore = false;
+
+    const fetchReleaseHistory = async () => {
+      let res: HelmRelease[];
+      try {
+        res = await coFetchJSON(`/api/helm/release/history?ns=${namespace}&name=${releaseName}`);
+      } catch {
+        if (ignore) return;
+      }
+      if (ignore) return;
+      res.length > 0 && setReleaseHistory(res);
```

Comment:
If there is an error, `res` will be undefined and then `res.length` will fail.

Also, the `if (ignore) ...` in the catch isn't needed if it's checked again immediately afterwards.

Reply:
Done.